jerry-core/vm/vm.c

```diff
@@ -153,15 +153,15 @@ vm_init (ecma_compiled_code_t *program_p) /**< pointer to byte-code data */
 /**
  * Decode table for opcodes.
  */
-static const uint32_t vm_decode_table[] =
+static const uint16_t vm_decode_table[] =
```

Comment:
No blocker but nasty: this change does not seem to be related to the patch at all. I'd love to see separate ideas in separate patches/PRs.